crates/raito-bridge-node/src/rpc.rs

```diff
@@ -52,13 +59,20 @@ impl RpcServer {
     async fn run_inner(&self) -> Result<(), std::io::Error> {
         info!("Starting RPC server on {}", self.config.rpc_host);
 
-        let app = Router::new()
+        let inclusion = Router::new()
             .route("/block-inclusion-proof/:block_height", get(generate_proof))
             .route("/head", get(get_head))
             .route("/roots", get(get_roots))
             .with_state(self.app_client.clone())
             .layer(TraceLayer::new_for_http());
 
+        let compressed = Router::new()
```

Comment:
Why a separate router?

Reply:
I see, it's just a bit weird that you query block inclusion proof via local rpc rather than from the state directly